apps/mux/package.json

```diff
@@ -1,61 +1,26 @@
 {
-  "name": "@contentful/mux-contentful-uploader",
-  "version": "1.10.14",
+  "name": "@contentful/mux-app",
+  "version": "1.0.0",
   "private": true,
-  "devDependencies": {
-    "@contentful/app-scripts": "^1.2.0",
-    "@testing-library/jest-dom": "5.17.0",
-    "@testing-library/react": "12.1.5",
-    "@testing-library/user-event": "14.4.3",
-    "@types/jest": "28.1.7",
-    "@types/jsonwebtoken": "9.0.6",
-    "@types/node": "18.7.14",
-    "@types/react": "17.0.39",
-    "@types/react-dom": "17.0.11",
-    "@types/webpack-env": "1.18.4",
-    "@typescript-eslint/eslint-plugin": "^5.36.1",
-    "@typescript-eslint/parser": "^5.36.1",
-    "cross-env": "7.0.3",
-    "react-app-rewired": "^2.2.1",
-    "resize-observer-polyfill": "^1.5.1",
-    "typescript": "^4.7.4"
-  },
-  "dependencies": {
-    "@contentful/app-sdk": "4.12.0",
-    "@contentful/dam-app-base": "2.0.46",
-    "@contentful/f36-components": "4.61.0",
-    "@contentful/react-apps-toolkit": "1.2.16",
-    "@mux/mux-player-react": "2.3.2",
-    "@mux/mux-uploader-react": "1.0.0-beta.15",
-    "crypto-browserify": "^3.12.0",
-    "jsonwebtoken": "9.0.2",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-error-overlay": "6.0.11",
-    "react-scripts": "5.0.1",
-    "stream-browserify": "^3.0.0",
-    "util": "^0.12.5"
-  },
   "scripts": {
-    "start": "cross-env BROWSER=none react-app-rewired --openssl-legacy-provider start",
-    "build": "GENERATE_SOURCEMAP=false react-app-rewired --openssl-legacy-provider build",
-    "test": "react-app-rewired test",
-    "test:ci": "CI=true react-app-rewired test --runInBand",
-    "eject": "react-app-rewired eject",
-    "lint": "eslint --max-warnings=0 .",
-    "create-app-definition": "contentful-app-scripts create-app-definition"
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "build": "rimraf ./build && npm run build-frontend && npm run build-app-actions",
+    "build-app-actions": "cd app-actions && npm ci && cd .. && node build-actions.js",
+    "build-frontend": "cd frontend && npm ci --include=dev && BUILD_PATH=../build npm run build",
+    "build:dev": "rimraf ./build && npm run build-frontend && npm run build-app-actions:dev",
+    "build-app-actions:dev": "cd app-actions && npm ci && cd .. && NODE_ENV=development node build-actions.js",
+    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEV_TESTING_ORG_ID} --definition-id 2Z5q0FnYZjRN79MjTE4ofn --token ${TEST_CMA_TOKEN}",
```

Comment:
yes. we actually deleted the `deploy` script awhile back to prevent deploys. this is one of the frustrating points about JSON -- you can't comment! If that were possible we would have just commented out the old deploy script with a note which would have been much clearer.